Tuturial-Structure-Template.md

```diff
@@ -1,68 +1,178 @@
 
-**IMPORTANT**: The last prerequisite will always be a link to the _“How to get started with Agora”_ tutorial using the url:
-```
-https://www.agora.io/en/blog/how-to-get-started-with-agora?utm_source=medium&utm_medium=blog&utm_campaign=
-```
+This page shows you how to add this functionality to you Agora Client and security infrastructure. 
 
-Append your tutorial’s title to the end of the link above, with an underscore replacing any spaces in the title, as in this example:
+## About the tech
 
-- **Blog Title:** How to Build a Token Server for Agora Applications using GoLang
-- **URL**: https://www.agora.io/en/blog/how-to-get-started-with-agora?utm_source=medium&utm_medium=blog&utm_campaign=How_to_Build_a_Token_Server_for_Agora_Applications_using_GoLang
+> A helicopter view that explains the architecture and workflow that the user is implementing.
+> 
+> For example:
 
-It's imperative that you type this URL correctly as it will be the primary way Agora can properly track developer conversion from individual content posts. 
+To *achieve the task outlined in the Title* you use *these technologies* that *work in the following way*.
 
-## Project Setup ##
-> (use as section title)
-Describe the steps for creating a new project and setting up the environment. 
+>Add an architecture diagram, workflow diagram, animation, screenshot here.
+> 
+> For example:
 
-If there are any commands that need to be run in the command line make sure to include them using the “embedded code” feature ```  
+The following figure shows the callflow between the different elements you use to implement step-up-authentication for Agoras:
+![Step up authenticatino workflow](./images/step_up_authentication_workflow.png "Step up authenticatino workflow")
 
-## Build XYZ ##
-> (use as section title replace XYZ with project feature being built in this section)  
+> Add a concise but complete explanation of the task. Where needed, link to a trusted source
 
-Write the individual steps to building the specific feature within the project. 
 
-There will be multiple Build XYZ sections within a single tutorial. The goal is to modularize the various elements and build the project in a logical way that makes it easy for the user to follow along.
+## Prerequisites
 
-### Build Subsection ### 
-> (replace with title of subsection being built in this section)  
+> List what the reader needs to know, as well as packages or SDKS that must be available, to successfully work through the tutorial. Keep the list short, but be sure to include language and concepts the reader should be familiar with.
+> 
+> **IMPORTANT**: The first prerequisite will always include a link to the _“How to get started with Agora”_ tutorial using URL: https://www.agora.io/en/blog/how-to-get-started-with-agora?utm_source=medium&utm_medium=blog&utm_campaign=
```

Comment:
I suppose this applies to dev-rel writing only, or we can have a similar page in the document as well.
It is worth noting that contents in the blog post need update.

Reply:
Nope, everything. 

There is a discussion going on about tracking at the moment. This is a messy way to do it in the first place, and should apply to doc as well as blog posts. I have seen products where the main entry point was a get started in the doc rather than blog posts.

Reply:
Maybe we can link to a get-started doc instead of the blog?

Reply:
Fair. @digitallysavvy, are you ok with this?